### PR TITLE
doc: Fix manual

### PR DESCRIPTION
**Pull request by johannes-wolf**
Fixes the compilation of manual.typ.

### CODE REVIEW

File: src/lib.typ [hunk 1]
```diff
@@ -3,10 +3,10 @@
 #import "canvas.typ": canvas
 #import "draw.typ"
 
-// #import "coordinate.typ"
-// #import "vector.typ"
-// #import "matrix.typ"
-// #import "tree.typ"
+#import "styles.typ"
```
Comment (johannes-wolf) [inline src/lib.typ:6]: I want to have these exposed under `cetz.*`.

File: manual.typ [hunk 2]
```diff
@@ -315,7 +313,7 @@ line(a, b, c, stroke: gray)
 bezier-through(a, b, c, name: "b")
 
 // Show calculated control points
-line(a, "b.ctrl-1", "b.ctrl-2", c, stroke: gray)
+line(a, "b.ctrl-0", "b.ctrl-1", c, stroke: gray)
```
[the remark this answers is not in the document]
Comment (fenjalien) [reply manual.typ:316]: Its because `bezier`'s ctrl anchors start at 0.
https://github.com/johannes-wolf/cetz/blob/master/src/draw.typ#L990-L992

**Comment by fenjalien** [comment]
I would like to leave the manual/docs alone until we finish some more reworks as it seems a bit futile if we're just going to break things again...

**Comment by johannes-wolf** [comment]
> I would like to leave the manual/docs alone until we finish some more reworks as it seems a bit futile if we're just going to break things again...

Making it compile gives the benefit that changes to it from other PRs can be compiled.
I would actually like to merge this to have a state on 0.2.0 that at least is compilable now.

**Comment by fenjalien** [comment]
@johannes-wolf this needs rebasing

**Comment by johannes-wolf** [comment]
Rebased.

**Review by fenjalien — APPROVED** [review]

**Pull request merged**